sycl/include/CL/sycl/INTEL/esimd.hpp

```diff
@@ -10,17 +10,17 @@
 
 #pragma once
 
-/// \defgroup sycl_esimd DPC++ Explicit SIMD API
-
-#include <CL/sycl/INTEL/esimd/esimd.hpp>
-#include <CL/sycl/INTEL/esimd/esimd_math.hpp>
-#include <CL/sycl/INTEL/esimd/esimd_memory.hpp>
-#include <CL/sycl/INTEL/esimd/esimd_view.hpp>
-
 #ifdef __SYCL_DEVICE_ONLY__
 #define SYCL_ESIMD_KERNEL __attribute__((sycl_explicit_simd))
 #define SYCL_ESIMD_FUNCTION __attribute__((sycl_explicit_simd))
 #else
 #define SYCL_ESIMD_KERNEL
 #define SYCL_ESIMD_FUNCTION
 #endif
+
```

Comment:
This change is needed because `SYCL_ESIMD_FUNCTION` is used in the headers included below.